src/Microsoft.DotNet.Interactive/Server/StdioKernelKernelClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Threading.Tasks;
+
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Connection;
+using Microsoft.DotNet.Interactive.Events;
+
+namespace Microsoft.DotNet.Interactive.Server
+{
+    public class StdioKernelClient : KernelClientBase, IDisposable
```

Comment:
I thought the idea was to use the existing `KernelClient` so we don't keep adding implementations.